### ISSUE TITLE
Make the `master` branch to be protected after we got one more commiters

### ISSUE THREAD

**Issue by huan** [wechaty/scala-wechaty#10]
So that we can improve the code quality for our `master` branch.

To be done.

**Comment by jcai** [comment]
禁止force push?

**Comment by huan** [comment]
Yes. after the branch has been protected, the force push will be forbidden.

And there are also other benefits:

1. need to use Pull Request to merge code into the master branch
1. The CI must turn green
1. Requires code review & approval
